target/linux/lantiq/image/vr9.mk

```diff
@@ -201,6 +259,32 @@ define Device/avm_fritz7430
 endef
 TARGET_DEVICES += avm_fritz7430
 
+define Device/avm_fritz7490
+  $(Device/dsa-migration)
+  $(Device/AVM)
+  $(Device/NAND)
+  DEVICE_MODEL := FRITZ!Box 7490
+  DEVICE_VARIANT := Other NAND
+  KERNEL_SIZE := 4096k
+  IMAGE_SIZE := 49152k
+  DEVICE_PACKAGES := kmod-usb3 fritz-tffs kmod-avm-wasp kmod-avm-wasp \
+	-kmod-owl-loader -dsl-vrx200-firmware-xdsl-a -dsl-vrx200-firmware-xdsl-b-patch
+endef
+TARGET_DEVICES += avm_fritz7490
+
+define Device/avm_fritz7490_micron
```

Comment:
This should be `avm_fritz7490-micron` (replace underscore with dash) to be consistent with dts, same with the other boxes.

Otherwise, after booting initramfs, sysupgrade needs to be flashed using `-f`, because:
```
upgrade: Device avm,fritz7490-micron not supported by this image
upgrade: Supported devices: avm,fritz7490,micron Image check failed.
```

Reply:
I have changed that for all devices in this PR.